README.md

```diff
@@ -82,6 +82,21 @@ events.broadcast('post.created', user_id: 1)
 # => 'something created'
 ```
 
+### Formatters
+You can use different formatters for displaying information about all your events for event instance. Now hanami-events support:
+* plain text formatter
+* json formatter
+
+```ruby
+events.subscribe('*') { |payload| p 'all events' }
+events.subscribe('user.*') { |payload| p 'user events' }
+events.subscribe('*.created') { |payload| p 'something created' }
+
+require 'hanami/events/formatter'
+events.format(:json) # => JSON string with all events
```

Comment:
Could we show example of formatted strings here? To be more clear that it will return just event names